src/results/proforma.jl

```diff
@@ -111,6 +97,7 @@ function proforma_results(p::REoptInputs, d::Dict)
         # In the two party case the developer does not include the fuel cost in their costs
         # It is assumed that the offtaker will pay for this at a rate that is not marked up
         # to cover developer profits
+        # TODO escalate fuel cost with p.s.financial.generator_fuel_cost_escalation_rate_fraction
```

Comment:
@Bill-Becker you can remove this TODO now, right?

Reply:
Yes